File: net/JetBrains.SignatureVerifier/SignatureData.cs
namespace JetBrains.SignatureVerifier
{
  public readonly struct SignatureData
  {
    public SignatureData(byte[] signedData, byte[] cmsData)
    {
      SignedData = signedData;
      CmsData = cmsData;
    }

    public byte[] SignedData { get; }
    public byte[] CmsData { get; }
    public bool IsEmpty => SignedData is null;
  }
}

File: jvm/src/main/kotlin/com/jetbrains/signatureverifier/SignatureData.kt
package com.jetbrains.signatureverifier

class SignatureData(val SignedData: ByteArray?, val CmsData: ByteArray?) {
  val IsEmpty: Boolean
    get() = CmsData == null
  val HasAttachedSignedData: Boolean
    get() = SignedData != null

  companion object {
    @JvmStatic
    val Empty = SignatureData(null, null)
  }
}
